src/fusiontrees/manipulations.jl

```diff
@@ -767,7 +770,7 @@ function elementary_trace(f::FusionTree{I,N}, i) where {I<:Sector,N}
         return newtrees
     else # i == N
         if N == 2
-            f′ = FusionTree{I}((), one(I), (), (), ())
+            f′ = FusionTree{I}((), f.coupled, (), (), ()) # or leftone(f.uncoupled[1]) == rightone(f.uncoupled[2])
```

Comment:
I don't think this is right for reasons that I can explain on the blackboard. I think the coupled sector needs to be `leftone(b) = leftone(f.uncoupled[i]) = leftone(f.uncoupled[2])`, instead of `rightone(f.uncoupled[2])`. Maybe we can have `unit = leftone(b)` at the top of this `else # i == N` clause, and then use that `unit` also as the coupled sector of `f′` both here and below. I am not sure if even the special casing the `N==2` case is really necessary, but it helps a bit.